package.json

```diff
@@ -62,12 +62,14 @@
 		"build-css:editor": "check-npm-client && node-sass --importer node_modules/node-sass-package-importer/dist/cli.js --include-path client client/assets/stylesheets/editor.scss public/editor.css --output-style compressed && yarn run -s postcss public/editor.css",
 		"build-css:reader-mobile": "check-npm-client && node-sass --importer node_modules/node-sass-package-importer/dist/cli.js --include-path client client/assets/stylesheets/reader-mobile.scss public/reader-mobile.css --output-style compressed && yarn run -s postcss public/reader-mobile.css",
 		"build-devdocs:search-index": "check-npm-client && node bin/generate-devdocs-search-index.js",
-		"build-desktop": "check-npm-client && run-s build-desktop:config build-desktop:server build-desktop:client build-desktop:install-app-deps build-desktop:app",
+		"build-desktop": "check-npm-client && yarn run build-desktop:source && run-s build-desktop:install-app-deps build-desktop:app",
```

Comment:
Can we run all three `build-desktop:*` tasks inside a single `run-s`? It effectively does the same thing as `yarn run a && yarn run b && yarn run c`, waiting for one job to finish before starting the next one.

Reply:
Updated in 57921dd56cfb7705d9f51af83ff774bbc25fea38.